ts/src/pro/bybit.ts

```diff
@@ -67,6 +67,9 @@ export default class bybit extends bybitRest {
                 'watchTicker': {
                     'name': 'tickers', // 'tickers' for 24hr statistical ticker or 'tickers_lt' for leverage token ticker
                 },
+                'watchOrderBook': {
```

Comment:
@ttodua don't think we should do it at the options level, it depends on a lot of factors such as market type, channel used, etc